packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1667,24 +1671,24 @@ class NotifierCheckbox extends StatelessWidget {
 
 class CheckboxSetting extends StatelessWidget {
   const CheckboxSetting({
-    Key key,
-    @required this.notifier,
-    @required this.title,
+    Key? key,
+    required this.notifier,
+    required this.title,
     this.description,
     this.tooltip,
     this.onChanged,
     this.enabled = true,
   }) : super(key: key);
 
-  final ValueListenable<bool> notifier;
+  final ValueListenable<bool?> notifier;
```

Comment:
This should be `ValueNotifier<bool>` so we don't have to type cast later in the build method

Reply:
nope: NotifierCheckbox.onChanged requires nullable parameter

Reply:
We can still change it to be a ValueNotifier instead of a ValueListenable to eliminate the need to type cast